### DISCUSSION TITLE
verify failed: unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by Ivan-Dorofeev** [deepgram/community#191] [General help] [answered]
("line 38: <class 'aiohttp.client_exceptions.ClientConnectorCertificateError'> "  '- Cannot connect to host api.deepgram.com:443 ssl:True '  "[SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate "  "verify failed: unable to get local issuer certificate (_ssl.c:1002)')]")

Hi, I didn't find any solution to this problem.
I have token with "admin" role and use VPN.

Does anybody know how to solve it?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi Ivan, you'll need to update your certificate chain to trust the server. What operating system are you on?

If you're on Debian, you can try the command: `apt-get install ca-certificates`

There are also some MacOS Python solutions suggested on Stackoverflow to install certificates: https://stackoverflow.com/questions/52805115/certificate-verify-failed-unable-to-get-local-issuer-certificate

**Comment by rkoverov** [comment]
Hi, I also didn't find any solution to this problem.
I am encountering an error while attempting to start the streaming example.
WebSocketException in LiveClient.start: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007).

Could you please help me troubleshoot the issue?

The pre-recorded examples worked perfectly.
OS: Ubuntu v. 22.04.4 LTS.

Thanks in advance.
